app/notification_push.go

```diff
@@ -427,6 +443,77 @@ func DoesStatusAllowPushNotification(userNotifyProps model.StringMap, status *mo
 	return false
 }
 
+func (a *App) BuildFetchedPushNotificationMessage(postId string, userId string) (model.PushNotification, *model.AppError) {
+	msg := model.PushNotification{
+		Type:     model.PUSH_TYPE_ID_LOADED,
+		Category: model.CATEGORY_CAN_REPLY,
+		Version:  model.PUSH_MESSAGE_V2,
+	}
+
+	post, err := a.GetSinglePost(postId)
+	if err != nil {
+		return msg, err
+	}
+
+	channel, err := a.GetChannel(post.ChannelId)
+	if err != nil {
+		return msg, err
+	}
+
+	user, err := a.GetUser(userId)
+	if err != nil {
+		return msg, err
+	}
+
+	sender, err := a.GetUser(post.UserId)
+	if err != nil {
+		return msg, err
+	}
+
+	msg.PostId = post.Id
+	msg.RootId = post.RootId
+	msg.SenderId = post.UserId
+	msg.ChannelId = channel.Id
+	msg.TeamId = channel.TeamId
+
+	notification := &PostNotification{
+		post:    post,
+		channel: channel,
+		sender:  sender,
+	}
+
+	cfg := a.Config()
+	nameFormat := a.GetNotificationNameFormat(user)
+	channelName := notification.GetChannelName(nameFormat, user.Id)
+	senderName := notification.GetSenderName(nameFormat, *cfg.ServiceSettings.EnablePostUsernameOverride)
+
+	contentsConfig := *cfg.EmailSettings.PushNotificationContents
```

Comment:
there is no need to check for this, remember this will get trigger when `id_loaded` so you'll get `id_loaded` here.

I guess you need to build the whole message as if it is `full` content